src/main/java/adris/altoclef/tasks/misc/RavageDesertTemplesTask.java

```diff
@@ -0,0 +1,105 @@
+package adris.altoclef.tasks.misc;
+
+import adris.altoclef.AltoClef;
+import adris.altoclef.tasks.movement.SearchWithinBiomeTask;
+import adris.altoclef.tasks.squashed.CataloguedResourceTask;
+import adris.altoclef.tasksystem.Task;
+import adris.altoclef.util.ItemTarget;
+import net.minecraft.block.Blocks;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.biome.Biome;
+
+import java.util.List;
+
+public class RavageDesertTemplesTask extends Task {
+    private BlockPos _currentTemple;
+    private Task _lootTask;
+    private Task _pickaxeTask;
+    public final Item[] LOOT = {
+            Items.BONE,
+            Items.ROTTEN_FLESH,
+            Items.GUNPOWDER,
+            Items.SAND,
+            Items.STRING,
+            Items.SPIDER_EYE,
+            Items.ENCHANTED_BOOK,
+            Items.SADDLE,
+            Items.GOLDEN_APPLE,
+            Items.GOLD_INGOT,
+            Items.IRON_INGOT,
+            Items.EMERALD,
+            Items.IRON_HORSE_ARMOR,
+            Items.GOLDEN_HORSE_ARMOR,
+            Items.DIAMOND,
+            Items.DIAMOND_HORSE_ARMOR,
+            Items.ENCHANTED_GOLDEN_APPLE
+    };
+
+    public RavageDesertTemplesTask() {
+
+    }
+
+    @Override
+    protected void onStart(AltoClef mod) {
+        mod.getBehaviour().push();
+        mod.getBlockTracker().trackBlock(Blocks.STONE_PRESSURE_PLATE);
+    }
+
+    @Override
+    protected Task onTick(AltoClef mod) {
+        if (_lootTask != null && !_lootTask.isFinished(mod)) {
+            setDebugState("Looting found temple");
+            return _lootTask;
+        }
+        if (_pickaxeTask != null && !_pickaxeTask.isFinished(mod)) {
+            setDebugState("Need to get pickaxes first");
+            return _pickaxeTask;
+        }
+        if (!mod.getItemStorage().hasItemInventoryOnly(Items.WOODEN_PICKAXE) && !mod.getItemStorage().hasItemInventoryOnly(Items.STONE_PICKAXE)) {
+            setDebugState("Need to get pickaxes first");
+            _pickaxeTask = new CataloguedResourceTask(new ItemTarget(Items.WOODEN_PICKAXE, 2));
+            return _pickaxeTask;
+        }
+        _currentTemple = getADesertTemple(mod);
+        if (_currentTemple != null) {
+            _lootTask = new LootDesertTempleTask(_currentTemple, List.of(LOOT));
+            setDebugState("Looting found temple");
+            return _lootTask;
+        }
+        return new SearchWithinBiomeTask(Biome.Category.DESERT);
```

Comment:
Why not use `LocateDesertTempleTask`?

Reply:
What for?

Reply:
Still dont know what the point of using that task is

Reply:
u right actually lol it will do the same thing here